docs/pipelineruns.md

```diff
@@ -888,6 +888,26 @@ spec:
     finally: "0h3m0s"
 ```
 
+Combination 3: Set only a `tasks` timeout, with no timeout for the entire `pipeline`.
+
+```yaml
+kind: PipelineRun
+spec:
+  timeouts:
+    pipeline: "0"  # No timeout
+    tasks: "0h3m0s"
+```
+
+Combination : Set only a `finally` timeout, with no timeout for the entire `pipeline`.
+
+```yaml
+kind: PipelineRun
+spec:
+  timeouts:
+    pipeline: "0"  # No timeout
+    finally: "0h3m0s"
+```
+
 You can also use the *Deprecated* `timeout` field to set the `PipelineRun's` desired timeout value in minutes.
 If you do not specify this value in the `PipelineRun`, the global default timeout value applies.
 If you set the timeout to 0, the `PipelineRun` fails immediately upon encountering an error.
```

Comment:
thanks @lbernick for the fix!

Does this statement still hold true? This is documented in the next paragraph.

```
 If you set the timeout value or timeouts.pipeline to 0, the PipelineRun fails immediately upon encountering an error.
```